process_report/process_report.py

```diff
@@ -202,7 +154,7 @@ def main():
     parser.add_argument(
         "--output-file",
         required=False,
-        default="filtered_output.csv",
+        default="filtered_output",
```

Comment:
since we are at it, might as well give it a better default name like `billable_invoice`